### PR TITLE
Make portfolio graph loading more efficient

### PR DESCRIPTION
**Pull request by mqp**
Since the default view period is a week, it sucked to pull down the whole history.

Note that the old code sort of implied that there might be history for prior to June, but in the actual database it seems that history was removed. There was no mechanism for filtering stuff only past June, and at a glance, it seems like the history collections I can find don't contain pre-June data points. So e.g. the code that was showing the "since June" label was never doing anything.

I also re-enabled the "daily" option. Austin claimed in [this commit](https://github.com/manifoldmarkets/manifold/commit/258b2a318fa626df536ca31ce178ec6ea6af4919) that it's "broken" but I see zero evidence of that. I'm guessing he got confused by the thing where the metrics update job was busted for a few days due to OOM.

### CODE REVIEW

File: web/lib/firebase/users.ts [hunk 1]
```diff
@@ -252,11 +252,12 @@ export async function unfollow(userId: string, unfollowedUserId: string) {
   await deleteDoc(followDoc)
 }
 
-export async function getPortfolioHistory(userId: string) {
+export async function getPortfolioHistory(userId: string, since: number) {
   return getValues<PortfolioMetrics>(
     query(
       collectionGroup(db, 'portfolioHistory'),
       where('userId', '==', userId),
+      where('timestamp', '>=', since),
```
Comment (jahooma) [inline web/lib/firebase/users.ts:260]: Nice.

File: web/components/portfolio/portfolio-value-section.tsx [hunk 2]
```diff
@@ -50,24 +55,22 @@ export const PortfolioValueSection = memo(
...
+            className="select select-bordered self-start"
+            value={portfolioPeriod}
+            onChange={(e) => {
+              setPortfolioPeriod(e.target.value as Period)
+            }}
+          >
+            <option value="allTime">All time</option>
+            <option value="weekly">Last 7d</option>
+            {/* Note: 'daily' seems to be broken? */}
+            {/* <option value="daily">Last 24h</option> */}
```
Comment (jahooma) [inline web/components/portfolio/portfolio-value-section.tsx:68]: Did you mean to enable this option? I think it's a good idea.

**Review by jahooma — APPROVED** [review]
Looks good. Thanks Marshall!

File: web/components/portfolio/portfolio-value-section.tsx (hunk 2 above)
Comment (mqp) [reply web/components/portfolio/portfolio-value-section.tsx]: Oops, just forgot to push that commit.

**Pull request merged**